src/driver.toit

```diff
@@ -67,10 +77,15 @@ class Driver:
 
       value := binary.BIG_ENDIAN.int24 raw 0
       if count < 4: value >>= 8
-      return value / (1000 << resolution_ * 2).to_float
+      result := value / (1000 << resolution_ * 2).to_float
```

Comment:
Put parenthesis around the nested expression.
I never remember the precedence of `<<` vs `*`.
Also, I would convert the value to a float first. Just reads nicer.:
```suggestion
      result := value.to_float / (1000 << (resolution_ * 2))
```